WordPress/src/main/java/org/wordpress/android/ui/reader/usecases/ReaderCommentsFollowUseCase.kt

```diff
@@ -124,4 +155,38 @@ class ReaderCommentsFollowUseCase @Inject constructor(
 
         object UserNotAuthenticated : FollowCommentsState()
     }
+
+    private enum class AnalyticsFollowCommentsAction(val action: String) {
+        FOLLOW_COMMENTS("followed"),
+        UNFOLLOW_COMMENTS("unfollowed")
+    }
+
+    private enum class AnalyticsFollowCommentsActionResult(val actionResult: String) {
+        SUCCEEDED("succeeded"),
+        ERROR("error")
+    }
+
+    private enum class AnalyticsFollowCommentsGenericErrors(val actionError: String) {
```

Comment:
Should this be `Error` instead of `Errors`? And since we use `actionError` as an error message, what do you think about renaming it to `errorMessage` or something similiar?

Reply:
yep makes sense 👍 . Changed here 4744775